modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectValidationRuleResourceTest.java

```diff
@@ -212,6 +235,56 @@ protected ObjectValidationRule randomObjectValidationRule()
 				randomObjectValidationRule());
 	}
 
+	private void _unsafeTriConsumer(
```

Comment:
@pedro-oliveira446 rename to `_getPageWithSortStringUnsafeTriConsumer` methods should be named as actions.

Thanks!